packages/extractor-vue/src/__test__/test.vue

```diff
@@ -0,0 +1,32 @@
+<script setup>
+import { i18n } from "@lingui/core"
+import { defineComponent } from "vue"
+
+i18n._("Setup message")
+</script>
+
+<script lang="ts">
+export default defineComponent({
+  data() {
+    return {
+      i18n,
+      scriptString: i18n._("Script message"),
+    }
+  },
+})
+</script>
```

Comment:
add specific to TS only syntax, to validate that extractor will not fail with "unexpected token",  something simple, for example: 

```ts
const foo: Type = 5;
```